src/components/Modal/OfferDetailsModal.tsx

```diff
@@ -10,6 +10,13 @@ interface OfferDetailModalProps {
 }
 
 const OfferDetailsModal: React.FC<{ id: string }> = ({ id }) => {
+   function countDaysAgo(createdAt: string): number {
+      const createdAtDate = new Date(createdAt);
```

Comment:
**nitpick:** This should be a utility function, that is imported here. You would easily reuse such function in other places in the system